src/applications/vaos/tests/e2e/direct-schedule.e2e.spec.js

```diff
@@ -0,0 +1,114 @@
+const Timeouts = require('../../../../platform/testing/e2e/timeouts.js');
+const VAOSHelpers = require('./vaos-helpers');
+const Auth = require('../../../../platform/testing/e2e/auth');
+
+const continueButton = '.rjsf [type="submit"]';
+
+module.exports = {
+  after: (client, done) => {
+    client.deleteCookies();
+    client.end();
+    done();
+  },
+  login: client => {
+    const token = Auth.getUserToken();
+
+    VAOSHelpers.initAppointmentListMock(token);
+
+    Auth.logIn(
+      token,
+      client,
+      '/health-care/schedule-view-va-appointments/appointments/',
+      3,
+    ).waitForElementVisible('#appointments-list', Timeouts.slow);
+    // .axeCheck('.main');
+  },
+  'Select new appointment': client => {
+    client
+      .click('#new-appointment')
+      .waitForElementVisible(continueButton, Timeouts.normal)
+      .axeCheck('.main');
+  },
+  'Choose the type of care you need': client => {
+    client.click('[value="211"]').perform(() => {
+      client
+        .click('.rjsf [type="submit"]')
+        .waitForElementPresent('h1', Timeouts.normal)
+        .assert.containsText('h1', 'Choose a VA location for your appointment');
+    });
+  },
+  'Choose a VA location for your appointment': client => {
+    client.click('[value="983"]').perform(() => {
+      client
+        .waitForElementPresent(
+          '[name = "root_vaFacility"][value = "983"]',
+          Timeouts.normal,
+        )
+        .click('[name = "root_vaFacility"][value = "983"]')
+        .pause(Timeouts.normal)
```

Comment:
I think there's probably a way to get rid of this timeout, but it's tricky, so I'll create an issue to deal with it later.

Reply:
Please elaborate. I can work on it if you want.

Reply:
To get rid of it I think we'd need to wait for the continue to be enabled (it has a loading spinner for a bit after you choose the facility_, so you'd have to come up with a selector that worked for that to put in `waitForElementVisible`, and I'm not positive we can use complicated attribute selectors in Nightwatch. So there'd be some trial and error.

Reply:
If you want to take a crack at it for 30 minutes or so, that's fine with me, but I don't want to hold up the PR for a bunch more time for something I'm not positive will work.